packages/office-addin-project/package.json

```diff
@@ -11,6 +11,9 @@
     "prettier": "office-addin-lint prettier",
     "test": "mocha -r ts-node/register test/**/*.ts"
   },
+  "bin": {
+    "office-addin-project": "lib/cli.js"
```

Comment:
According to comments in other packages, a cli.js should be in the root that references the build js in the lib folder to be loaded dynamically.  This avoids problems when the lib folder doesn't exist.